Add 反切原貌

syimyuzya opened the pull request:
~~Still WIP.~~ Ready for merge!

Issues to be resolved:
- [x] Differences caused only by character variants
- [x] 反切s appearing in 釋義 are not in their original form, but with poem's corrections applied

syimyuzya committed:
Add 反切原貌

Issues to be resolved:
- Differences caused only by character variants
- 反切s appearing in 釋義 are not in their original form, but with
  poem's corrections applied.

syimyuzya committed:
Fix check.py to reflect format changes

syimyuzya commented:
關於「釋義」（及「釋義補充」）中出現的反切，與反切原貌或 NK2028 所校之反切不同的共有 78 筆，目前很不統一。具體有以下幾種情況：

- （51 筆）poem 修正過，且與咱們採用的修正一致。釋義中反切亦為修正後的（其中部分僅為異體導致的差異，待清點）
- （26 筆）poem 未修，但咱們修過反切。釋義中的反切尚為原貌
- （1 筆）poem 修正過，但其修正與咱們的不同。釋義中反切為 poem 版（導致釋義中的反切與「反切」「反切原貌」兩欄均不同）

syimyuzya commented:
> 關於「釋義」（及「釋義補充」）中出現的反切，與反切原貌或 NK2028 所校之反切不同的共有 78 筆，目前很不統一。具體有以下幾種情況：
> 
>     * （51 筆）poem 修正過，且與咱們採用的修正一致。釋義中反切亦為修正後的（其中部分僅為異體導致的差異，待清點）
> 
>     * （26 筆）poem 未修，但咱們修過反切。釋義中的反切尚為原貌
> 
>     * （1 筆）poem 修正過，但其修正與咱們的不同。釋義中反切為 poem 版（導致釋義中的反切與「反切」「反切原貌」兩欄均不同）

全部 78 筆有差異者（已按類型排序，上文所述三類分別對應下表之 ABB、AAB、ABC）：

```csv
類型,小韻號,音韻地位,反切原貌,反切poem校,反切NK2028校,首字,首字釋義
AAB,209,溪開三C嚴入,丘之,丘之,丘乏,抾,挹也丘之切一
AAB,399,昌三尤平,昌來,昌來,昌求,㹗,昌來切牛羊無子一
AAB,409,匣開四先上,下珍,下珍,下殄,礥,鞭也下珍切又下憐切三
AAB,646,,丁全,丁全,丁兮,𡰝,行不正皃丁全切一
AAB,1032,清合三脂平,千侯,千侯,千隹,䜅,就也千侯切一
AAB,1040,精開三之平,子幽,子幽,子絲,稵,禾生也子幽切一
AAB,1041,心開一覃平,山幽,山幽,山函,犙,牛三歲山幽切一
AAB,1191,清三鍾上,職勇,職勇,且勇,幒,說文曰㡓也職勇切二
AAB,1307a,初開三臻入,初紀,初紀,初乙,㓼,割聲初紀切三
AAB,1307b,初合二夬去,初紀,初紀,初夬,欼,齧也
AAB,1420,見開二山入,丈夥,丈夥,丈黠,㧳,攙㧳㧳物出聲譜丈夥切一
AAB,1422,見開二山入,花夥,花夥,花黠,扮,亂扮也花夥切一
AAB,1423a,定開二佳上,求蟹,求蟹,乖蟹,箉,竹具用之魚笱竹器也求蟹切二
AAB,1487,日合三仙上,而允,而允,而兖,蝡,淮南子曰蠉飛蝡動或作蠕而允切又而兗切一
AAB,1574,從合三虞上,辝纂,辝纂,辝矩,鄹,字林云亭名在新豐辝纂切一
AAB,1692a,以合三脂上,以沼,以沼,以水,鷕,雉鳴也以沼切又羊水切七
AAB,2004,溪開三B侵上,士㾕,士㾕,丘㾕,𩒣,𩒣𩖄醜皃士㾕切一
AAB,2060,見開三A幽上,兼玷,兼玷,居黝,孂,竦身皃兼玷切一
AAB,2340,心開三仙上,思句,思句,思典,尠,思句切少也又息淺切一
AAB,2381,徹開三魚平,丑戾,丑戾,丑居,𥱻,竹名也丑戾切又杖胡切一
AAB,2561,從開三真去,徐刃,徐刃,疾刃,賮,琛賮又財貨也會禮也徐刃切又疾刃切六
AAB,2767a,曉開三C蒸入,火弔,火弔,火即,㱇,說文云悲意也火弔切三
AAB,2991a,曉開一覃平,許令,許令,許含,𣢝,含笑也許令切二
AAB,3645,匣合二耕入,簪摑,簪摑,于馘,㩇,裂聲也簪摑切一
AAB,3671,曉合三A清入,七役,七役,火役,𥄎,小動七役切二
AAB,3854,疑開四青上,五夾,五夾,五冷,䀴,埤蒼云視皃五夾切一
ABB,133,精合三支平,姊宜,姊規,姊規,厜,厜㕒山巔狀姊規切六
ABB,157,書開三脂平,式之,式脂,式脂,尸,主也陳也利也又姓秦有尸佼爲商君師著書式脂切四
ABB,179,昌合三脂平,叉隹,尺隹,尺隹,推,排也尺隹切又尺湯回切二
ABB,236,生開三魚平,所𦵔,所葅,所葅,疏,通也除也分也遠也窻也又姓漢有太子太傅東海疏廣或作𤕟俗作疎所葅切又所助切十一
ABB,353,影合二皆平,乙皆,乙乖,乙乖,崴,崴㠢乙乖切四
ABB,357,孃開二皆平,諧皆,諾皆,諾皆,搱,諾皆切一
ABB,401,章開三真平,側鄰,職鄰,職鄰,真,真僞也又姓風俗通云漢有太尉長史真祐俗作真職鄰切十六
ABB,463,滂三C文平,府文,撫文,撫文,芬,芬芳又姓戰國策晉有大夫芬質撫文切十三
ABB,500,滂一魂平,普魂,普䰟,普䰟,濆,潠也普䰟切三
ABB,1097,精開一談平,昨三,作三,作三,𩈻,長面皃作三切一
ABB,1155,並三C凡平,符咸,符䒦,符䒦,凡,常也皆也輕也非一也又姓周公子凡伯之後姓苑云晉陵人符䒦切七
ABB,1313,溪開三C微上,袪狶,袪豨,袪豨,豈,安也焉也曾也袪豨切二
ABB,1314,見開三C微上,居狶,居豨,居豨,蟣,蟣蝨居豨切四
ABB,1625,清開三仙上,士演,七演,七演,淺,不深也七演切一
ABB,1659,滂三B仙上,被免,披免,披免,鴘,埤蒼云鷹鷂二年色披免切一
ABB,1865,明二庚上,莫幸,莫杏,莫杏,猛,勇猛又嚴也害也惡也亦姓左傳宋大夫猛獲之後莫杏切六
ABB,1873,匣合二庚上,呼䁝,乎䁝,乎䁝,卝,金玉未成器也乎䁝切二
ABB,1986,書開三侵上,式任,式荏,式荏,沈,國名古作邥亦姓出吳興本自周文王第十子聃季食采於沈即汝南平輿沈亭是也子孫以國爲氏式荏切又丈林切十四
ABB,2062,疑開三C嚴上,魚掩,魚埯,魚埯,儼,敬也說文曰昂頭也一曰好皃魚埯切七
ABB,2109,清三東去,子仲,千仲,千仲,趥,行皃千仲切一
ABB,2284,並三C微去,扶涕,扶沸,扶沸,𥝋,獸名說文曰周成王時州靡國獻𥝋人身反踵自笑笑即上脣掩其目食人北方謂之土螻爾雅曰狒狒如人被髮迅走一名梟羊俗謂之山都今交州南康山中有之郭璞讚云狒狒怪獸披髮握竹獲人則笑脣蔽其目終乃號咷反爲我戮扶沸切二十四
ABB,2295,溪開三C魚去,近倨,丘倨,丘倨,㰦,欠去丘倨切九
ABB,2360,精一模去,臧𧙓,臧祚,臧祚,作,造也臧祚切一
ABB,2464,崇開二佳去,七懈,士懈,士懈,㾹,疾也士懈切二
ABB,2490,知合二皆去,他怪,迍怪,迍怪,顡,顏惡也迍怪切說文五怪切癡顡不聰明也一
ABB,2491,見合二夬去,古賣,古邁,古邁,夬,決也亦卦名古邁切三
ABB,2507,崇開二夬去,犲夬,豺夬,豺夬,寨,羊栖宿處豺夬切二
ABB,2534,溪開一咍去,苦蓋,苦愛,苦愛,慨,慷慨苦愛切六
ABB,2537,匣開一咍去,胡槩,胡概,胡概,瀣,沆瀣氣也胡概切三
ABB,2545,滂三C廢去,方廢,芳廢,芳廢,肺,金藏芳廢切四
ABB,2607,初合三元去,芳万,叉万,叉万,𣀔,小舂也亦作䊲叉万切一
ABB,2648,從開一寒去,祖贊,徂贊,徂贊,㜺,不謹也一曰美好皃徂贊切五
ABB,2662,幫一寒去,博慢,博漫,博漫,半,物中分也博漫切七
ABB,2741,崇合三仙去,七戀,士戀,士戀,䉵,說文曰具食也士戀切九
ABB,2750,以開三仙去,于線,予線,予線,衍,水也溢也豐也予線切又以淺切八
ABB,2807,崇開二肴去,七稍,士稍,士稍,巢,棧閣也士稍切又士交切一
ABB,2817,幫一豪去,博耗,博秏,博秏,報,報告下婬曰報博秏切一
ABB,2911,精開三陽去,于亮,子亮,子亮,醬,說文作酱醢也漢書武帝使唐蒙風曉南越南越食蒙蜀蒟醬子亮切四
ABB,3067,定一侯去,田候,徒候,徒候,豆,穀豆物理論云菽者眾豆之名也又姓後魏有將軍豆代田徒候切十六
ABB,3167,孃開二咸去,尼賺,尼𧸖,尼𧸖,諵,尼𧸖切一
ABB,3313,明三B真入,美畢,美筆,美筆,密,說文云山脊也又靜也亦州名古姑幕城秦琅邪郡隋爲密州因水以名之又姓漢有尚書密忠又漢複姓三氏何氏姓苑云密茅氏琅邪人又有密革氏密須氏俗作密美筆切十
ABB,3352,見開三C殷入,居乙,居乞,居乞,訖,止也居乞切五
ABB,3406,匣開一寒入,矛割,予割,予割,䔾,菜似蕨生水中予割切一
ABB,3442,疑合二山入,五骨,五滑,五滑,𦤙,屈也五滑切三
ABB,3501,溪開三B仙入,丘謁,丘竭,丘竭,朅,說文去也丘竭切又去謁切五
ABB,3534,常合三仙入,姝雪,殊雪,殊雪,啜,說文曰甞也爾雅曰茹也禮曰啜菽飲水殊雪切一
ABB,3535,初合三仙入,廁列,廁別,廁別,㔍,割斷聲廁別切一
ABB,3539,崇開三仙入,土列,士列,士列,𨵊,城門中板也士列切一
ABB,3791,清開一覃入,士合,七合,七合,遪,裹遪七合切一
ABB,3795,端開一談入,都搕,都榼,都榼,㿴,𥀰㿴都榼切十一
ABB,3845,崇開二咸入,七洽,士洽,士洽,箑,行書士洽切六
ABC,1478,云合三B真上,士忍,七忍,于忍,笉,笑皃七忍切一
```

syimyuzya committed:
Fix 釋義補充 for 菱 & 䔖 (again)

untunt commented:
## 校勘標記的方案

| 類型 | 小韻例子 | 純文本標記 | 線性顯示方案一 | 線性顯示方案二 |
| - | -: | - | - | - |
| 脫字 | 3067 豆 | [徒]候 | ${\rlap{\textcolor{YellowGreen}{\large\textsf■}}徒候}$ | $\textcolor{YellowGreen}[徒\textcolor{YellowGreen}]候$ |
| 訛字 | 1625 淺 | 士<七>演 | $\textcolor{Red}{\small士\textsf➜}七演$ | ${\textcolor{Red}士{\small\textsf⟨七\textsf⟩}演}$ |
| 改用其他來源的音韻地位 | 133 厜 | 姊宜⦉規⦊ | $姊\textcolor{Red}{\small宜\textsf➜}規$ | ${姊\textcolor{Red}宜{\small\textsf(規\textsf)}}$ |
| 替換成近似等價字，反切結果改變 | 1155 凡 | 符咸(䒦) | $符\textcolor{Red}{\small咸\textsf➜}䒦$ | ${符\textcolor{Red}咸{\small\textsf(䒦\textsf)}}$ |
| 替換成音近字，反切結果改變 | 157 尸 | 式之(脂) | $式\textcolor{Red}{\small之\textsf➜}脂$ | ${式\textcolor{Red}之{\small\textsf(脂\textsf)}}$ |
| 替換成等價字，反切結果不變 | 2918 放 | 甫⦅府⦆妄 | $\textcolor{RoyalBlue}{\small甫\textsf➜}府妄$ | ${\textcolor{RoyalBlue}甫{\small\textsf(府)}妄}$ |
| 替換成同音字，反切結果不變 | 32 烘 | 呼東⦅紅⦆ | $呼\textcolor{RoyalBlue}{\small東\textsf➜}紅$ | ${呼\textcolor{RoyalBlue}東{\small\textsf(紅)}}$ |
| 複合使用 | 1692a 鷕 | 以沼⦅小⦆<水> | $以\textcolor{RoyalBlue}{\small沼\textsf➜}\textcolor{Red}{\small小\textsf➜}水$ | ${以\textcolor{RoyalBlue}沼\textcolor{Red}{\small\textsf(小)}{\small\textsf⟨水⟩}}$ |

以 HTML 顯示時，應使校勘標記和校正前的字不能被選中。在目前資料中 `⦅ ⦆` 僅在複合的情況（作爲中間步驟）使用。

根據校勘標記整合反切如下。

## 異體（8 筆）

簡言之：236 有新的修改意見，500、2537 恢復原貌，其他與現在資料相同。

| 類型 | 小韻號 | 音韻地位    | 首字 | 反切 | 說明                                                                                         |
| ---- | ------ | ----------- | ---- | ---- | -------------------------------------------------------------------------------------------- |
| ABB  | 236    | 生開三魚平  | 疏   | 所菹 | 澤存堂本下字作“𦵔”，poem改作“葅”，今按菹小韻字頭（菹……亦作葅）改作“菹”               |
| ABB  | 500    | 滂一魂平    | 濆   | 普魂 | 澤存堂本下字作“魂”，poem改作“䰟”，但䰟小韻釋義中也多用“魂”，因此不必改                 |
| ABB  | 1313   | 溪開三C微上 | 豈   | 袪豨 | 澤存堂本下字作“狶”，按豨小韻字頭（豨……亦作狶）改作“豨”                                 |
| ABB  | 1314   | 見開三C微上 | 蟣   | 居豨 | 澤存堂本下字作“狶”，按豨小韻字頭（豨……亦作狶）改作“豨”                                 |
| ABB  | 2507   | 崇開二夬去  | 寨   | 豺夬 | 澤存堂本上字作“犲”，按豺小韻字頭（未給出異體）改作“豺”                                   |
| ABB  | 2537   | 匣開一咍去  | 瀣   | 胡槩 | 澤存堂本下字作“槩”，poem改作“槩”，但溉小韻槩字字頭就作“槩”（poem改作“概”），實不必改 |
| ABB  | 2817   | 幫一豪去    | 報   | 博秏 | 澤存堂本下字作“耗”，按秏小韻字頭（秏……俗作耗）改作“秏”                                 |
| ABB  | 3167   | 孃開二咸去  | 諵   | 尼𧸖 | 澤存堂本下字作“賺”，按𧸖小韻字頭（未給出異體）改作“𧸖”                                   |

## 非異體（71 筆）

| 類型 | 小韻號 | 音韻地位    | 首字 | 反切                 |
| ---- | ------ | ----------- | ---- | -------------------- |
| ABB  | 133    | 精合三支平  | 厜   | 姊宜⦉規⦊           |
| ABB  | 157    | 書開三脂平  | 尸   | 式之(脂)             |
| ABB  | 179    | 昌合三脂平  | 推   | 叉<尺>隹             |
| AAB  | 209    | 溪開三C嚴入 | 抾   | 丘之<乏>             |
| ABB  | 353    | 影合二皆平  | 崴   | 乙皆(乖)             |
| ABB  | 357    | 孃開二皆平  | 搱   | 諧<諾>皆             |
| AAB  | 399    | 昌三尤平    | 㹗   | 昌來<求>             |
| ABB  | 401    | 章開三真平  | 真   | 側(職)鄰             |
| AAB  | 409    | 匣開四先上  | 礥   | 下珍<殄>             |
| ABB  | 463    | 滂三C文平   | 芬   | 府(撫)文             |
| AAB  | 646    |             | 𡰝   | 丁全<兮>             |
| AAB  | 1032   | 清合三脂平  | 䜅   | 千侯<隹>             |
| AAB  | 1040   | 精開三之平  | 稵   | 子幽<絲>             |
| AAB  | 1041   | 心開一覃平  | 犙   | 山幽<函>             |
| ABB  | 1097   | 精開一談平  | 𩈻   | 昨<作>三             |
| ABB  | 1155   | 並三C凡平   | 凡   | 符咸(䒦)             |
| AAB  | 1191   | 清三鍾上    | 幒   | 職⦉且⦊勇           |
| AAB  | 1307a  | 初開三臻入  | 㓼   | 初紀⦅己⦆<乙>       |
| AAB  | 1307b  | 初合二夬去  | 欼   | 初紀⦅史⦆<夬>       |
| AAB  | 1420   | 見開二山入  | 㧳   | 丈<?>夥<黠>          |
| AAB  | 1422   | 見開二山入  | 扮   | 花<?>夥<黠>          |
| AAB  | 1423a  | 定開二佳上  | 箉   | 求<?>蟹              |
| AAB  | 1423b  | 見合二佳上  | 拐   | 求<乖>蟹             |
| ABC  | 1478   | 云合三B真上 | 笉   | 士<于>忍             |
| AAB  | 1487   | 日合三仙上  | 蝡   | 而允<兖>             |
| AAB  | 1574   | 從合三虞上  | 鄹   | 辝纂⦅短⦆<矩>       |
| ABB  | 1625   | 清開三仙上  | 淺   | 士<七>演             |
| ABB  | 1659   | 滂三B仙上   | 鴘   | 被<披>免             |
| AAB  | 1692a  | 以合三脂上  | 鷕   | 以沼⦅小⦆<水>       |
| ABB  | 1865   | 明二庚上    | 猛   | 莫幸(杏)             |
| ABB  | 1873   | 匣合二庚上  | 卝   | 呼<乎>䁝             |
| ABB  | 1986   | 書開三侵上  | 沈   | 式任<荏>             |
| AAB  | 2004   | 溪開三B侵上 | 𩒣   | 士⦅仕⦆<丘>㾕       |
| AAB  | 2060   | 見開三A幽上 | 孂   | 兼⦅居⦆玷⦅點⦆<黝> |
| ABB  | 2062   | 疑開三C嚴上 | 儼   | 魯<魚>掩<埯>         |
| ABB  | 2109   | 清三東去    | 趥   | 子<千>仲             |
| ABB  | 2284   | 並三C微去   | 𥝋   | 扶涕<沸>             |
| ABB  | 2295   | 溪開三C魚去 | 㰦   | 近<丘>倨             |
| AAB  | 2340   | 心開三仙上  | 尠   | 思句⦅具⦆<典>       |
| ABB  | 2360   | 精一模去    | 作   | 臧𧙓<祚>             |
| AAB  | 2381   | 徹開三魚平  | 𥱻   | 丑戾<居>             |
| ABB  | 2464   | 崇開二佳去  | 㾹   | 七<士>懈             |
| ABB  | 2490   | 知合二皆去  | 顡   | 他<迍>怪             |
| ABB  | 2491   | 見合二夬去  | 夬   | 古賣(邁)             |
| ABB  | 2534   | 溪開一咍去  | 慨   | 苦蓋(愛)             |
| ABB  | 2545   | 滂三C廢去   | 肺   | 方<芳>廢             |
| AAB  | 2561   | 從開三真去  | 賮   | 徐(疾)刃             |
| ABB  | 2607   | 初合三元去  | 𣀔   | 芳⦅反⦆<叉>万       |
| ABB  | 2648   | 從開一寒去  | 㜺   | 祖<徂>贊             |
| ABB  | 2662   | 幫一寒去    | 半   | 博慢<漫>             |
| ABB  | 2741   | 崇合三仙去  | 䉵   | 七<士>戀             |
| ABB  | 2750   | 以開三仙去  | 衍   | 于<予>線             |
| AAB  | 2767a  | 曉開三C蒸入 | 㱇   | 火弔<即>             |
| ABB  | 2807   | 崇開二肴去  | 巢   | 七<士>稍             |
| ABB  | 2911   | 精開三陽去  | 醬   | 于<子>亮             |
| AAB  | 2991a  | 曉開一覃平  | 𣢝   | 許⦅火⦆令<含>       |
| ABB  | 3067   | 定一侯去    | 豆   | [徒]候               |
| ABB  | 3313   | 明三B真入   | 密   | 美畢(筆)             |
| ABB  | 3352   | 見開三C殷入 | 訖   | 居乙<乞>             |
| ABB  | 3406   | 匣開一寒入  | 䔾   | 矛<予>割             |
| ABB  | 3442   | 疑合二山入  | 𦤙   | 五骨<滑>             |
| ABB  | 3501   | 溪開三B仙入 | 朅   | 丘謁<竭>             |
| ABB  | 3534   | 常合三仙入  | 啜   | 姝<殊>雪             |
| ABB  | 3535   | 初合三仙入  | 㔍   | 廁列<別>             |
| ABB  | 3539   | 崇開三仙入  | 𨵊   | 土<士>列             |
| AAB  | 3645   | 匣合二耕入  | 㩇   | 簪⦅子⦆<于>摑       |
| AAB  | 3671   | 曉合三A清入 | 𥄎   | 七<火>役             |
| ABB  | 3791   | 清開一覃入  | 遪   | 士<七>合             |
| ABB  | 3795   | 端開一談入  | 㿴   | 都搕<榼>             |
| ABB  | 3845   | 崇開二咸入  | 箑   | 七<士>洽             |
| AAB  | 3854   | 疑開四青上  | 䀴   | 五夾⦅洽⦆<冷>       |

特別的說明：

| 類型 | 小韻號 | 音韻地位    | 首字 | 反切           | 說明                                                                                          |
| ---- | ------ | ----------- | ---- | -------------- | --------------------------------------------------------------------------------------------- |
| AAB  | 1420   | 見開二山入  | 㧳   | 丈<?>夥<黠>    | nk2028資料作“丈黠”，今進一步將上字改爲“?”（不明）                                         |
| AAB  | 1422   | 見開二山入  | 扮   | 花<?>夥<黠>    | nk2028資料作“花黠”，今進一步將上字改爲“?”（不明）                                         |
| AAB  | 1423b  | 見合二佳上  | 拐   | 求<?>蟹        | nk2028資料作“乖蟹”，修了跟沒修一樣，今將上字改爲“?”（不明）                               |
| AAB  | 1423b  | 見合二佳上  | 拐   | 求<乖>蟹       | 前面評論的 78 筆差異遺漏此条                                                                  |
| ABC  | 1478   | 云合三B真上 | 笉   | 士<于>忍       | poem校作“七忍”，不取                                                                        |
| ABB  | 2062   | 疑開三C嚴上 | 儼   | 魯<魚>掩<埯>   | 澤存堂本作“魯掩”，資料中反切原貌作“魚掩”不準確                                            |
| AAB  | 2991a  | 曉開一覃平  | 𣢝   | 許⦅火⦆令<含> | nk2028資料作“許含”，今進一步將上字改爲舌根和諧的“火”，同 1083 㟏（火含切，內有“𣢺”字） |
| ABB  | 3067   | 定一侯去    | 豆   | [徒]候         | 澤存堂本脫“徒”字，資料中反切原貌作“田候”不準確，“田”實爲上一句話末字                    |
| AAB  | 3645   | 匣合二耕入  | 㩇   | 簪⦅子⦆<于>摑 | nk2028資料作“于馘”，下字不應校正爲“馘”，“馘”只出現於“又于馘切”                        |

此外有一些與布之道《廣韻形聲考》的差異：

- 409 礥、1422 扮、3671 𥄎《考》尚未收錄
- 1420 㧳【丈<?>夥<黠>】《考》作“丈<火>夥”，係採用 unt 稍早觀點
- 2004 𩒣【士⦅仕⦆<丘>㾕】《考》作“士<牛>㾕”，誤作“𩖄”之音
- 2060 孂【兼⦅居⦆玷⦅點⦆<黝>】《考》作“兼玷⦅點⦆<黝>”，上字也應還原爲舌根和諧的“居”

## 新增修改（6 筆）

| 類型 | 小韻號 | 音韻地位                 | 首字 | 反切               | 說明                                                       |
| ---- | ------ | ------------------------ | ---- | ------------------ | ---------------------------------------------------------- |
| AAA  | 138    | 章合三支平               | 䮔   | 子⦅足⦆<之>垂     | 以前沒有反切原貌功能，故不敢改反切。現在可改               |
| AAA  | 400    | 滂一灰平 > 滂三C尤上     | 𡜊   | 普才⦅來⦆<求>     | 据《考》新增                                               |
| AAA  | 569    | 來合三支平               | 𡰠   | 力頑<規>           | 以前沒有反切原貌功能，故不敢改反切。現在可改               |
| AAA  | 1830   | 並三A陽上 > 並三A宵平    | 𩦠   | 毗養⦅兩⦆<霄>     | 以前沒有反切原貌功能，故不敢改反切。現在可改，地位也一併改 |
| AAA  | 2767c  | 曉開三C蒸入 > 澄開三仙入 | 娎   | 火<丈>弔⦅叫⦆<列> | 据《考》新增。2767b原轄“嬈、娎”二字，“娎”拆出成2767c   |
| AAA  | 3062   | 清三尤去 > 清合一灰平    | 䞭   | 七溜⦅霤⦆<雷>     | 据《考》新增                                               |

[之前拆分小韻](https://github.com/nk2028/tshet-uinh-data/issues/4#issuecomment-2094699578)的原則是僅在小韻**首字**音韻地位有誤時才拆分，以節省工作量。如果依此原則，2767b 就暫可不拆；但既然 2767（原轄“㱇、嬈、娎”三字）已經拆分成 2767a、2767b，那麼不妨拆到底。

最後，經過整理，布之道《廣韻形聲考》（2024.4.26 版）中還有 104 條反切校正待引入，但由於它們不是小韻首字，所以依原則暫不引入，它們也幾乎不涉及常用字。其中，7 條是只轄兩字的小韻的後一字，但也暫不引入。（今天整理過程中新發現的 400 𡜊、2767c 娎、3062 䞭已加入上表）

untunt commented:
## 新增修改補充

| 類型 | 小韻號 | 音韻地位 | 首字 | 反切 | 說明 |
| -- | -- | -- | -- | -- | -- |
| AAA | 1763 | 清開一歌上 | 硰 | 作⦅子⦆<千>可 | 《漢書注》“師古曰：硰音千坐反”（一作“子坐反”）。諸家注音皆爲清母，《集韻》亦删 tsaq 而留 tshaq。tsaq 雖不非法，但是應當改。以前只改了音韻地位，現在反切也改 |
| AAA | 3670 | 見合三A清入 > 章合三清入 | 𦳮 | 之⦅志⦆<?>役 | 《P.5531》志役反。“志”一般不作上字，有誤。之前推斷爲“吉”或“古”之訛，但是“吉”也一般不作上字，“古”不切三等韻。暫讓音韻地位回退 |

untunt commented:
音韻地位和反切的調整「合集」可參照 [QYS4MCPDict 代碼](https://github.com/untunt/QYS4MCPDict/blob/782a949e70a4ea9490bd3c9187b40951174c95bf/deriver_consts.js#L174)（注意其中將 `<>` 換成了 `⟨⟩`）

untunt commented:
再次更新：小韻 1628（蹇謇搴𢷘𠐻𢵈𦂇𡾰藆𩽜䙭；見開三B仙上；九輦切）拆分爲 1628a（蹇謇搴𢷘𠐻𦂇𡾰藆𩽜䙭；見開三B仙上）和 1628b（𢵈；見開三A仙上）兩小韻

理由：

> 《韻鏡》三四等皆爲蹇；《七音略》三等爲蹇，四等無字；《五音集韻》在三等。
>
> 《切三 S.2071》《P.3693》爲「蹇、謇、搴」（居輦反）和「𢵈」（基善反）兩小韻，《王三》《王一 P.2011》亦爲兩小韻且給蹇小韻增「𦂇」字。《廣韻》《集韻》合併。
>
> 「𢵈」以外的字爲 B 類沒有問題，但「𢵈」的類不明（諧聲亦不明，不管）。
>
> 《篆隸萬象名義》「搌：刄〈丑〉善反。𢵈丶（字/也），展極。𢵈：絶〈紀〉善反。言〈展〉極丶（也）。」
>
> 宋本《玉篇》「𢵈：紀善切。搌𢵈，醜長皃。搌：丑善切。搌𢵈。」
>
> 曹憲《博雅音》「搌：展。𢵈：膳。呂靜音已〈己〉善切。」（《廣雅》「搌𢵈，展極也。」）
>
> 《玉篇》系字書反切下字「善」_djienq_ 切鈍音聲母三等時均爲 A 類，如《玉篇》殘卷「䭤：去善反」_khienq_、「遣：丘善反」_khienq_、「繾：祛善反」_khienq_、「顈：呼善反」_hwienq_（《集韻》馨兖切），《篆隸萬象名義》「𣝜：扶善反」_bienq_（《集韻》婢善切）、「沔：弥善反」_mienq_、「𩋠：亡善反」_mienq_、「緬：弭善反」_mienq_。
>
> 由此，「基善反」應理解爲 A 類 _kienq_，《廣韻》小韻應拆分。

----

同時，因「類」而拆分的小韻，反切也應當用 ( ) 標註（表示近似等價字），以使反切與音韻地位一致：

| 小韻號 | 代表字 | 廣韻反切     | 附：切韻反切 | 音韻地位描述 |
| ------ | ------ | ------------ | ------------ | ------------ |
| 1043a  | 飍     | 香幽         | 香幽         | 曉開三A幽平  |
| 1043b  | 烋     | 香⦅許⦆幽(彪) | 許彪         | 曉開三B幽平  |
| 1628a  | 蹇     | 九輦         | 居輦         | 見開三B仙上  |
| 1628b  | 𢵈     | 九輦(善)     | 基善         | 見開三A仙上  |
| 3521a  | 孑     | 居列         | 居列         | 見開三A仙入  |
| 3521b  | 揭     | 居列(?)      | (無此小韻)   | 見開三B仙入  |
| 3708a  | 憶     | 於力         | 於力         | 影開三C蒸入  |
| 3708b  | 抑     | 於力(棘)     | 於棘         | 影開三B蒸入  |

untunt commented:
再再次更新：

- 566 㺗 充山切：初開二山平→昌開三仙平
  > 「單」聲不應讀莊組，《集韻》增加 _tjhien_ 音。
- 1281 𧿲 止姊切：章開三脂上→莊開三之上
  > 「𠂔」聲不應讀章組，《五音集韻》增加 _tsryq_ 音。

反切均無需更改。

syimyuzya committed:
wip: use new 小韻 data for 反切 & 音韻地位

- 反切 now includes annotations for representing original and corrected
  forms
- 音韻地位 updated accordingly
  - Every 小韻 now has a 音韻地位

TBD:
- Fix 反切 in 釋義 and 釋義補充
- Add missing checks back

syimyuzya committed:
Restore original 反切 in 釋義

TBD:
- Check 反切

syimyuzya committed:
Check 反切 & 釋義

syimyuzya commented:
已加入並更新以上所有反切及音韻地位（並與音典新資料對照無誤）。小韻首字釋義中的反切亦從 poem 所校反切還原為（調整碼位後的）原貌。

如果之後還有需要的更動，可以在新回覆中寫～

syimyuzya committed:
Correct 小韻 141 𤿎: 匹支(之)切

滂三A支平 → 滂三A脂平

syimyuzya commented:
#10 將對本 PR 做出的額外改進：

- 字頭「概」調整為「槩」，與反切整理用字一致
- 「𤜼」「㶒」「𣅝」三個無效小韻給出字頭說明
